Dockerfile

```diff
@@ -1,29 +1,55 @@
-FROM golang:1.13 as secrets-provider-builder
+# =================== BASE BUILD LAYER ===================
+# this layer is used to prepare a common layer for both debug and release builds
+FROM golang:1.13 as secrets-provider-builder-base
 MAINTAINER CyberArk Software Ltd.
 
 ENV GOOS=linux \
     GOARCH=amd64 \
     CGO_ENABLED=0
 
-COPY . /opt/secrets-provider-for-k8s
+RUN go get -u github.com/jstemmer/go-junit-report && \
```

Comment:
This seems like it should be installed during test runs and not included in the container we ship out

Reply:
Added it as a consideration for a following issue #150